### DISCUSSION TITLE
[Suggestion] Increase search result rankings in VSCode

### DISCUSSION THREAD

**Discussion by QuocAnhVu** [pest-parser/pest#831] [Q&A] [answered]
When I first was looking for the vscode extension, I simply searched "pest" and picked the first/most starred option. I actually had no idea there was an official extension until I read the navbar at pest.rs more closely.

I can tell for myself: if it the extension was instead named "Pest", I would be more likely to look into it as an option instead of having it been lost in noisy search results. ("IDE Tools" here is superfluous. All vscode extensions are IDE tools.) I also think the description should also contain the word "official" and link to pest.rs in the description. Finally, we can always nicely ask @Stef-Gijsberts to change the name of his/her extension :)

![Screenshot from 2023-04-06 12-55-40](https://user-images.githubusercontent.com/5098329/230481321-21ee22e2-61f6-4554-b97b-f782d7a25821.png)

**Comment by Stef-Gijsberts** [comment, marked as answer]
I will gladly make changes to make the distinction between the official extension and mine more clear. I could call it 'Pest Syntax Highlighter'?

Another option is to completely remove my extension if all of its functionality is encapsulated in the official extension (I was not aware of it's existence). I'm not looking for double work :)

EDIT: I see now that my extension was [integrated into pest ide-tools](https://github.com/pest-parser/pest-ide-tools#credits), cool! Thanks for the credits. I will think for a bit if I can come up with reasons to keep my extension alive, otherwise I will archive my repository and remove or rename the extension.

**Reply by tomtau** [reply]
@Jamalam360

**Comment by Jamalam360** [comment]
Yes the search rankings are annoying, I can see what we can do :)
